plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -460,6 +462,8 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
       await taskTrack.markFailed(step, err);
       await stepTrack.markFailed();
       throw err;
+    } finally {
+      await task.serializeWorkspace?.({ path: workspacePath });
```

Comment:
I wonder if we should put this behind a feature toggle for now. `scaffolder.experimental.workspaceSerialization` or something, just so that if we run into some performance issues we can turn it off/on.

Reply:
Good point.
Also added a cleanup of workspace if the task was successfully executed.